addons/fortify/stringtable.xml

```diff
@@ -173,14 +173,16 @@
             <German>Zeit-Kosten-Koeffizient</German>
             <Japanese>時間コスト係数</Japanese>
             <Polish>Współczynnik Czas-Koszt</Polish>
+            <Russian>Коэф. время-цена</Russian>
         </Key>
         <Key ID="STR_ACE_Fortify_settingHintDesc_timeCostCoefficient">
-            <English>Coefficient used to determine time to build structure.&lt;br/&gt;A in Ax + b where x is cost of object</English>
```

Comment:
this PR fixes
![20221202185824_1](https://user-images.githubusercontent.com/9376747/205414991-b834535d-e863-4670-ba66-ba31541f697a.jpg)